docs/testbed/README.testbed.VsSetup.md

```diff
@@ -138,6 +139,18 @@ mkdir -p ~/veos-vm/images
 mv sonic-vs.img ~/veos-vm/images
 ```
 
+### Option 2: Download sonic-vpp image
+1. Download the sonic-vs image from [here](https://github.com/cisco-mt-fuji/Sonic-Vpp).
+Note that the link is currently a private repo for internal use only, we will update the url with latest successful build soon.
```

Comment:
we can reference the sonic-platform-vpp for guidance of building the image.

Reply:
Addressed @r12f .